api/src/org/labkey/api/data/TableInfo.java

```
@@ -606,11 +606,11 @@ void fireRowTrigger(Container c, User user, TriggerType type, boolean before, in
      */
     default boolean hasDbTriggers() { return false; }
 
-    /**
-     * TableInfos that can be associated with a QueryHelper need a reliable key other than a TableInfo instance.
-     * Return null if QueryHelper is not supported.
-     */
-    Path getNotificationKey();
+    @Deprecated // No longer used
+    default Path getNotificationKey()
+    {
+        return null;

```

If we can't completely delete this method, should it throw `UnsupportedOperationException`?

---

I plan to delete this method in a quick follow-up. I just didn't feel like adding the WNPRC and LIMS repos to this branch.